backend/molgenis-emx2-fairdatapoint/src/main/java/org/molgenis/emx2/fairdatapoint/FAIRDataPointCatalog.java

```diff
@@ -225,6 +233,13 @@ Optional in FDP specification (https://specs.fairdatapoint.org/)
     this.result = stringWriter.toString();
   }
 
+  private static void nullCheckOnPropVal(String[] propertyValueSplit) throws Exception {
+    if (propertyValueSplit.length != 2) {
+      throw new Exception(
```

Comment:
Can't you throw a more specific Exception

Reply:
yep